src/components/molecules/password-field/index.js

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+
+import icInvisible from '_images/ic-invisible.png'
+import icVisible from '_images/ic-visible.png'
+
+import Label from '../../atoms/label'
+import Input from '../../atoms/input'
+
+import classes from './styles.css'
+
+class passwordMask extends Component {
+  state = {
+    type: 'password',
+  }
+
+  showHide = event => {
+    event.preventDefault()
+    event.stopPropagation()
+    this.setState({
+      type: this.state.type === 'input' ? 'password' : 'input',
```

Comment:
I'm a bit confused with this logic, I think if you use a `bool` instead of `string` we will be more readable.

```es6
this.setState((prevState) => ({
     isRawInput: !prevState.isRawInput
}))
```

and add this logic in the render function.